homeassistant/components/zwave_me/number.py

```diff
@@ -0,0 +1,54 @@
+"""Representation of a switchMultilevel."""
+from datetime import timedelta
+import logging
+
+from homeassistant.components.number import NumberEntity
+from homeassistant.helpers.dispatcher import async_dispatcher_connect
+
+from .__init__ import ZWaveMeDevice
+from .const import DOMAIN
+
+SCAN_INTERVAL = timedelta(seconds=10)
+
+_LOGGER = logging.getLogger(__name__)
+DEVICE_NAME = "switchMultilevel"
+
+
+async def async_setup_entry(hass, config, add_entities, discovery_info=None):
+    """Set up the sensor platform."""
+
+    def add_new_device(new_device):
+        switch = ZWaveMeNumber(new_device)
+        add_entities(
+            [
+                switch,
+            ]
+        )
+
+    async_dispatcher_connect(
+        hass, "ZWAVE_ME_NEW_" + DEVICE_NAME.upper(), add_new_device
+    )
```

Comment:
Track the unsub of this function in the config entry and use f-strings.
```suggestion

    entry.async_on_unload(
        async_dispatcher_connect(
            hass, f"ZWAVE_ME_NEW_{DEVICE_NAME.upper()}", add_new_device
        )   
    )
```